Makefile

```
@@ -1,20 +1,16 @@
 
-ROOTCFLAGS   := $(shell root-config --cflags)
-ROOTLIBS     := $(shell root-config --libs)
-
 CXX           = g++
 CXXFLAGS      = -Wall -fPIC -O3 -D CARTESIAN
 LD            = g++
 LDFLAGS       = -O3
 
-CXXFLAGS     += $(ROOTCFLAGS)
-LIBS          = $(ROOTLIBS) $(SYSLIBS)
+LIBS          = $(SYSLIBS) -lgsl
```

I (fedora 41 or 42) cannot build the executable unless I add this flag, seems to be a problem with linking gsl libraries. I am not sure this is a universally good change for different systems. @yukarpenko, should this go in? 

```suggestion
LIBS          = $(SYSLIBS) -lgsl -lgslcblas
```

---

with the ` -lgslcblas` addition it compiles on my Ubuntu as well.